deepmd/pt/train/training.py

```diff
@@ -247,7 +249,10 @@ def get_sample():
         def get_single_model(
             _model_params,
         ):
-            model = get_model(deepcopy(_model_params)).to(DEVICE)
+            if "use_srtab" in _model_params:
+                model = get_zbl_model(deepcopy(_model_params)).to(DEVICE)
+            else:
+                model = get_model(deepcopy(_model_params)).to(DEVICE)
```

Comment:
Maybe we use only one func called get_model, and wrap all these judgement into get_model.